variables.tf

```diff
@@ -14,6 +14,10 @@ variable "subnet_id" {
 #
 # These parameters have reasonable defaults.
 # ------------------------------------------------------------------------------
+variable "ami_owner_account_id" {
```

Comment:
You need a type here.

Reply:
Thanks- fixed in https://github.com/cisagov/skeleton-tf-module/pull/39/commits/1b84540513882933410a958e9205fb11cf8094dc.